overlord/overlord_test.go

```diff
@@ -1399,3 +1400,25 @@ func (ovs *overlordSuite) TestLockWithTimeoutFailed(c *C) {
 		"EXTEND_TIMEOUT_USEC=5000",
 	})
 }
+
+func (ovs *overlordSuite) TestEnsureLoopLogging(c *C) {
+	entries, err := os.ReadDir(".")
+	c.Assert(err, IsNil)
+
+	for _, entry := range entries {
+		if entry.IsDir() && strings.HasSuffix(entry.Name(), "state") {
+			testFilePath := filepath.Join(entry.Name(), entry.Name()+"_test.go")
+
+			if _, err := os.Stat(testFilePath); err == nil {
+				mgrFiles, err := filepath.Glob(filepath.Join(entry.Name(), "*mgr.go"))
+				c.Assert(err, IsNil)
+
+				if len(mgrFiles) > 0 {
+					content, err := os.ReadFile(testFilePath)
+					c.Assert(err, IsNil)
+					c.Assert(strings.Contains(string(content), "TestEnsureLoopLogging"), Equals, true, Commentf("File %s does not contain a TestEnsureLoopLogging test."))
```

Comment:
The comment is missing an argument for the `%`
```suggestion
					c.Assert(strings.Contains(string(content), "TestEnsureLoopLogging"), Equals, true, Commentf("File %s does not contain a TestEnsureLoopLogging test.", testFilePath))
```